src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -384,6 +402,7 @@ public BanditResult getBanditAction(
       Actions actions,
       String defaultValue) {
     BanditResult result = new BanditResult(defaultValue, null);
+    final Configuration config = configurationStore.getConfiguration();
     try {
       String assignedVariation =
           getStringAssignment(
```

Comment:
🐛 (but not introduced in this PR): `getStringAssignment` might get different `Configuration`, so `banditKeyForVariation` might fail unexpectedly. "obfuscated" in metadata might also be incorrect for this reason. This is an existing/old bug

Reply:
worth addressing. I filed an issue in linear